concentration/Scene/SpinningBoxAndBalls.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  SpinningBoxAndBalls.swift
+//  concentration
+//
+//  Created by Olha Dzhyhirei on 1/30/23.
+//
+
+import SpriteKit
+
+class SpinningBoxAndBalls: SKScene {
+    
+    var BallQuantity = 16
+    
+    enum GameState { case startGame, playing }
+    var gameState = GameState.startGame
+    var box: SKShapeNode!
+    
+    override func didMove(to view: SKView) {
+        self.anchorPoint = .init(x: 0, y: 0)
+        self.scene?.backgroundColor = .link
+        setupStartLabel()
+    }
+    
+    // MARK:  game state: startGame
+    
+    func setupStartLabel() {
+        let startLabel = SKLabelNode()
+        startLabel.name = "start label"
+        startLabel.fontColor = .black
+        startLabel.fontSize = 26
+        startLabel.text = "Tap anywhere to start"
+        startLabel.fontName = "AvenirNext-Bold"
+        startLabel.zPosition = 2
+        startLabel.position = .init(x: self.frame.midX, y: self.frame.midY)
+        let action = SKAction.sequence([.scale(by: 1.25, duration: 1), .scale(by: 0.8, duration: 1)])
+        startLabel.run(.repeatForever(action))
+        self.addChild(startLabel)
+        
+    }
+    
+    // MARK: game state: setupPlaying
+    
+    func createBall(ofRadius radius: CGFloat) -> SKSpriteNode {
+        let ball = SKSpriteNode(texture: .init(imageNamed: "ball_04"))
+        ball.size = .init(width: radius * 2, height: radius * 2)
+        ball.physicsBody = .init(circleOfRadius: radius)
+        ball.physicsBody?.isDynamic = false
+        ball.physicsBody?.restitution = 1
```

Comment:
make this property about 0.1-0.5 so they don't jump like crazy :)

Reply:
done